src/openrct2/park/ParkFile.cpp

```diff
@@ -1932,6 +1932,10 @@ namespace OpenRCT2
         cs.ReadWrite(entity.seat_rotation);
         cs.ReadWrite(entity.target_seat_rotation);
         cs.ReadWrite(entity.IsCrashedVehicle);
+        if (os.GetHeader().TargetVersion >= 17)
+        {
+            cs.ReadWrite(entity.isReversed);
```

Comment:
I'm not a huge fan of having this saved, why would you actually need to store that when you can always query the operating mode on the ride?

Reply:
IMO a per-vehicle property is ideal. It allows plugins like vehicle editor to manipulate the state of individual vehicles so it's not limited to rides with the operating mode.

Reply:
While I agree with the sentiment on plugins having the ability to change things I would argue that exposing individual members is not ideal, wouldn't it be better if plugins could describe an operating mode as a whole model rather? I haven't looked too close on how the whole thing works together but we might be able to create something that allows to describe all sorts of operating modes with data alone.

Reply:
What would "describing an operating mode as a whole model" entail? This sounds like something outside the scope of this PR. Regardless of if this PR implements API for it, it should be separated from the operating mode because OpenRCT2 users in the Deurklink and NEDesigns communities like to hack vehicles and tying it to operating mode would remove that utility.

Reply:
Since it looks like the checkbox implementation is the way to go, I'll mark this convo resolved - unless anyone has anything else to add